src/commands/createComponent.ts

```diff
@@ -19,11 +20,13 @@ export async function createComponent(
   }
 
   const targetPath = workspace.workspaceFolders[0].uri.toString().replace("file://", "");
-  const outFile = `${targetPath}/${name}`;
+  const outFile = `${targetPath}/component-archive`;
 
-  const cmd = `ocm create componentarchive ${name} ${version} --provider ${provider} --scheme ${scheme} -o ${outFile}`;
+  const cmd = `ocm create componentarchive ${name} ${version} --provider ${provider} --scheme ${scheme} -F ${outFile}`;
   output.send(`Creating component ${name}:${version} to ${outFile}...`);
 
   const result = await shell.exec(cmd);
-  output.send(`Component creation complete: ${result.stdout}`);
+  output.send(`Component creation complete: ${result.stdout || result.stderr}`);
```

Comment:
This prints `:` regardless of whether there is any output from the exec cmd, which looks a bit iffy.